pkg/ccl/backupccl/testdata/backup-restore/restore-schema-only-mixed-version

```diff
@@ -0,0 +1,32 @@
+# Tests that RESTORE with schema_only cannot get run in a mixed version cluster
+
+new-server name=s1 beforeVersion=34
```

Comment:
nit: can you comment what version 34 is just for easier readability.

Reply:
Done.